Services/SystemServer/main.cpp

```diff
@@ -84,6 +85,86 @@ static void parse_boot_mode()
     dbg() << "Booting in " << g_boot_mode << " mode";
 }
 
+static void prepare_devfs()
+{
+    // FIXME: Find a better way to all of this stuff, without hardcoding all of this!
+
+    int rc = mount(-1, "/dev", "dev", 0);
+    if (rc != 0) {
+        ASSERT_NOT_REACHED();
+    }
+    rc = mkdir("/dev/pts", 0755);
+    if (rc != 0) {
+        ASSERT_NOT_REACHED();
+    }
```

Comment:
Can we avoid doing this in userspace? Can DevFS create this automatically instead?

Reply:
I talked with Sergey about this, and considering the fact that we already allow `SystemServer` to use `chown`, (so DevFS isn't read-only anymore), it was an idea to avoid doing such things in the kernel code also.
But sure, if you think we should do that in the kernel space, it can be done without any trouble :)